besu/src/main/java/org/web3j/tx/PrivateTransactionManager.java

```diff
@@ -74,15 +74,15 @@ public PrivateTransactionManager(
             final Base64String privateFrom,
             final Base64String privacyGroupId,
             final Restriction restriction,
-            final TxSignService txSignService) {
+            final PrivateTxSignServiceImpl privateTxSignService) {
```

Comment:
Would this not cause API changes that might require this to be a major upgrade?

Reply:
Fixed this, should not break this API